src/main/scala/com/wavesplatform/Coordinator.scala

```diff
@@ -129,8 +137,8 @@ object Coordinator extends ScorexLogging with Instrumented {
     height = history.height()
     discardedTxs <- blockchainUpdater.processBlock(block)
   } yield {
-    if (local) BlockStats.mined(block, height) else BlockStats.applied(block, height)
-    TxsInBlockchainStats.record(block.transactionData.size - discardedTxs.size)
+    if (local) BlockStats.mined(block, height)
```

Comment:
we have 2 booleans which come from 2 sources:
1) miner: local = true, extension = false
2) recieved: local = false, extension = false
3) extension: local = false, extension = true

I think those who call this method are able to handle each case by themselves w/o propogating its info down here, the callee can just `.right.map` and do the logging